src/Core/Application/Dto/Manifest/TaskManifestRequestDto.php

```diff
@@ -39,28 +39,36 @@ class TaskManifestRequestDto implements ManifestRequestDtoInterface
      */
     protected ManifestFile $manifestFile;
```

Comment:
why do we need these DTO in Core? and also [AbstractRegistry.php](https://github.com/spryker-sdk/sdk/pull/275/files#diff-4ba94c3f02c3d5a32eb7b87590b47b5e8fa356ae45b3d7edc01889f1bebfbc90)118 and interfaces three no logic and you use it in infrastucture

Reply:
All the incoming DTOs should be in application layer to decouple the application from the different presentation types (CLI, REST)